src/functions/avl-processor/index.ts

```diff
@@ -1,3 +1,4 @@
+import { MetricUnits, Metrics } from "@aws-lambda-powertools/metrics";
```

Comment:
What's the reason for adding this package instead of using the existing `putMetricData` function from `src/shared/cloudwatch.ts`?